ts/main.ts

```diff
@@ -1 +1,55 @@
 /* global data */
+
+interface FormElements extends HTMLFormControlsCollection {
+  photoUrl: HTMLInputElement;
+  title: HTMLInputElement;
+  notes: HTMLTextAreaElement;
+}
+
+const $image = document.getElementsByTagName('img');
+const imf = $image[0] as HTMLImageElement;
+const $newImg = document.getElementById('photo-url') as HTMLInputElement;
+const $form = document.querySelector('#form') as HTMLFormElement;
+
+function imgChanger(event: Event): void {
+  const $change = event.target as HTMLInputElement;
+  const picture = $change.value;
```

Comment:
Minor thing but I would make these more descriptive than `$change` and `picture`. The first could just be your `$eventTarget` and the second could be `$eventTargetUrl` or something like that.